src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/InfiniteScrollList.xaml.cs

```diff
@@ -399,6 +417,11 @@ private Visibility EvaluateStatusBarVisibility(IItemLoader<PackageItemListViewMo
             return statusBarVisibility;
         }
 
+        /// <summary>
+        /// Appends <c>packages</c> to the <c>Items</c>
+        /// </summary>
+        /// <param name="packages">View model collection to add</param>
```

Comment:
terminology is weird...  you are adding all the items from the view model collection to the items collection.
can you make clearer?

Reply:
Rephrased the docs for this method.